composer.json

```diff
@@ -13,14 +13,18 @@
         {
             "name": "Jérémy Marodon",
             "email": "marodon.jeremy@gmail.com"
+        },
+        {
+            "name": "Peter Lee",
+            "email": "peter279k@gmail.com"
         }
     ],
     "require": {
         "php": "^5.6 || ^7.0",
-        "fabpot/goutte": "^3.1"
+        "guzzleHttp/guzzle": "^6.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.4.3",
+        "phpunit/phpunit": "^5.5",
```

Comment:
Nop because in the 5.4.3 there's an important fix in the library concerning namespaces